src/response.ts

```diff
@@ -0,0 +1,33 @@
+import idpayguard from './js/idpayguard';
+import { initHeader } from './js/header';
+import { setTranslateBtns } from './js/translateui';
+
+// eslint-disable-next-line sonarjs/cognitive-complexity
+document.addEventListener('DOMContentLoaded', async () => {
+  // TO UNCOMMENT
+  // document.body.classList.add('loadingOperations');
```

Comment:
the purpose of this `class` it to show `loadingOperations` e remove it on successufull PM `transaction` check, is it correct ?